src/test/java/api/loans/LoanAPITests.java

```diff
@@ -1621,6 +1690,23 @@ private void loanHasExpectedProperties(JsonObject loan) {
       loan.containsKey("itemStatus"), is(false));
   }
 
+  private void hasProperty(String property, JsonObject resource, String type, Object value) {
+    assertThat(String.format("%s should have an %s: %s",
+      type, property, resource),
+      resource.getMap().get(property), equalTo(value));
+  }
+
+  private void hasNoBorrowerProperties(JsonObject loanJson) {
+    hasNotProperty("userId", loanJson, "loan");
+    hasNotProperty(LoanProperties.BORROWER, loanJson, "loan");
+  }
+
+  private void hasNotProperty(String property, JsonObject resource, String type) {
```

Comment:
This name does not read easily for me. I've suggested an alternative what do you think.

```suggestion
  private void doesNotHaveProperty(String property, JsonObject resource, String type) {
```

Reply:
Agree